### PR TITLE
Update poetry recommendations & versions

### PR DESCRIPTION
**Pull request by jack-mcivor**

### CODE REVIEW

**Comment by ywolff** [comment]
Thanks, but Poetry 1.7 was released the day after you pushed this pull request 😄 
So, may as well go straight to 1.7, right?

**Comment by ywolff** [comment]
By the way, to avoid having to manually upgrade Poetry and other versions, I plan to use something like [renovate](https://github.com/renovatebot/renovate) in the long term.

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Install [pyenv](https://github.com/pyenv/pyenv) to manage your Python versions a
 
 Install [Poetry](https://python-poetry.org) to manage your dependencies and tooling configs:
   ```bash
-  curl -sSL https://install.python-poetry.org | python - --version 1.5.1
+  curl -sSL https://install.python-poetry.org | python - --version 1.6.1
```
Comment (ywolff) [inline README.md:48]: Upgrade to 1.7

**Review by ywolff — CHANGES_REQUESTED** [review]

**Comment by jack-mcivor** [comment]
@ywolff done

**Review by ywolff — APPROVED** [review]
Thanks @jack-mcivor!

**Pull request merged**